content/manual-references.json

```diff
@@ -1583,5 +1583,54 @@
   "publisher": "StatPearls", 
   "URL": "https://pubmed.ncbi.nlm.nih.gov/32310382",
   "PMID": "32310382"
+},
+{
+  "id": "url:https://www.nytimes.com/interactive/2021/us/covid-cases.html",
+  "type": "webpage",
+  "title": "Coronavirus in the U.S.: Latest Map and Case Count",
+  "container-title": "The New York Times",
+  "URL": "https://www.nytimes.com/interactive/2021/us/covid-cases.html",
+  "language": "en",
+  "accessed": {
+    "date-parts": [
+      [
+        "2022",
+        3,
+        11
+      ]
+    ]
+  }
+},
+{
+  "id": "https://www.washingtonpost.com/health/interactive/2022/omicron-comparison-cases-deaths-hospitalizations",
```

Comment:
Not intentional! I'll make another commit to try to fix that and some of the other DOI issues.

Reply:
Note that I think Manubot now requires wrapping URLs that end with `/` in the curly braces `{ }` when citing them in the text.  Otherwise they are not considered part of the identifier.

Reply:
Ooh I wonder if that would work as a better fix! This was failing because the trailing "/" is required to pull up the article.